po/sv.po

```diff
@@ -1683,14 +1671,14 @@ msgid "`%s' is not a valid key command"
 msgstr "`%s' är inte ett giltigt tangentkommando"
 
 #: src/keymap.cpp:764
-#, fuzzy, c-format
+#, c-format
 msgid "failed to parse operation sequence for %s"
-msgstr "Fel vid inläsning av webbflöde `%s': %s"
+msgstr ""
```

Comment:
That translation somehow got lost completely.

Reply:
The previous translation was completely wrong (it said "Error while reading feed"), but I wasn't happy with the translation I could come up with, so I just removed the previous translation. I should have mentioned that - I apologize. I'll fix it in a coming PR.